test/parallel/test-stream-duplex-destroy.js

```diff
@@ -284,3 +284,17 @@ const assert = require('assert');
   duplex.on('close', common.mustCall());
   duplex[Symbol.asyncDispose]().then(common.mustCall());
 }
+
+(async () => {
+  // Check Symbol.asyncDispose implicitly
+  await using duplex = new Duplex({
+    write(chunk, enc, cb) { cb(); },
+    read() {},
+  });
+  let count = 0;
+  duplex.on('error', common.mustCall((e) => {
+    assert.strictEqual(count++, 0); // Ensure not called twice
```

Comment:
`mustCall` already checks for that, we have `common.mustCallAtLeast` for when a function can be called an arbitrary amount of times
```suggestion
```